### ISSUE TITLE
Release Cluster Autoscaler 1.2.2, 1.1.3 and 1.0.6

### ISSUE THREAD

**Issue by aleksandra-malinowska** [kubernetes/autoscaler#857]
If you believe something should be included that isn't yet merged to relevant branch, please link a PR with cherry-pick here. Short description of why it's needed would be welcome.

**Comment by aleksandra-malinowska** [comment]
cc @feiskyer

**Comment by feiskyer** [comment]
@aleksandra-malinowska Thanks. 

https://github.com/kubernetes/autoscaler/pull/861 cherry picks #813 and #852 to cluster-autoscaler-release-1.2, which adds support of AKS.

**Comment by ReillyProcentive** [comment]
FYI, it looks like 1.1.3 and 1.0.6 weren't released successfully. They don't show up on the releases page and weren't uploaded to the Google container registry.

**Comment by aleksandra-malinowska** [comment]
@ReillyProcentive you're correct, they're in staging at the moment. 1.2.2 was released and included in 1.10.3. The reason for this is that patch releases for K8s 1.8 and 1.9 are much less frequent. Sorry for not communicating about it better, we're still working on improving the process here!
